packages/template-retail-react-app/app/pages/account/profile.jsx

```diff
@@ -275,24 +275,20 @@ const PasswordCard = () => {
                             isClosable: true
                         })
                         login.mutate({
-                            email: values.email,
+                            username: email,
                             password: values.password
                         })
                         passwordHeading?.focus()
+                        form.reset()
+                    },
+                    onError: async (err) => {
+                        const resObj = await err.response.json()
+                        form.setError('root.global', {type: 'manual', message: resObj.detail})
                     }
                 }
             )
-            setIsEditing(false)
-            toast({
-                title: formatMessage({
-                    defaultMessage: 'Password updated',
-                    id: 'password_card.info.password_updated'
-                }),
-                status: 'success',
-                isClosable: true
-            })
```

Comment:
Not a blocking comment, but I notice we're changing the UX and no longer showing a "✅  success" message. Does it not work with the altered functionality?

Reply:
This was a duplicate that was erroneously being shown either when there was an error or for a second time if it succeeded